resources/js/ApiPageConnectionRepo.js

```diff
@@ -152,9 +152,9 @@ module.ApiPageConnectionRepo = ( function ( mw, ApiConnectionsBuilder ) {
 				let fileIcons = [];
 				let fileSearch = [];
 				pages.forEach(function(page) {
-					if (page.pageprops && page.pageprops.titleicons) {
+					if (page.titleicons && page.titleicons) {
```

Comment:
Looks wrong

Reply:
Ah, yes, that is redundant since there is only a single level now. I will fix.

Reply:
Fixed. Thanks.

Reply:
Funny how many times I looked at that and thought it looked just fine.

Reply:
Mutation tests are definitely more reliable at catching this type of issues than us monkeys

Reply:
Definitely. It would probably be good to add some tests for the API results to test for things like the absence of pageprops, imageinfo, and titleicons. But, I'm not sure how to do that in JavaScript. Other than that, I have tested this patch manually both with and without TitleIcon installed, and it should be good to go.